pyproject.toml

```diff
@@ -1,3 +1,69 @@
+[project]
+name = "khal"
+dynamic = ["version"]
+description = "A standards based terminal calendar"
+readme = "README.rst"
+authors = [
+  {name = "khal contributors", email = "khal@lostpackets.de"},
+]
+license = {file = "LICENSE"}
+classifiers = [
```

Comment:
**nitpick, non-blocking**: What do you think of sorting these alphabetically? This usually avoids too much diff-noise, but this might be a good opportunity.